<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlbumCreator</class>
 <widget class="QMainWindow" name="AlbumCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>8192</width>
    <height>8192</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AlbumCreator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>8192</width>
     <height>6134</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget1">
       <layout class="QVBoxLayout" name="sourceImagePane">
        <item>
         <layout class="QHBoxLayout" name="pathSelectionPane">
          <item>
           <widget class="QPushButton" name="parentDirButton">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>16</height>
             </size>
            </property>
            <property name="toolTip">
             <string extracomment="change to parent directory"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="albumcreator_gui.qrc">
              <normaloff>:/:/images/1uparrow.png</normaloff>:/:/images/1uparrow.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pathField">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="FileBrowserListView" name="fileBrowser">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>160</height>
           </size>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::IgnoreAction</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>100</width>
            <height>75</height>
           </size>
          </property>
          <property name="movement">
           <enum>QListView::Snap</enum>
          </property>
          <property name="resizeMode">
           <enum>QListView::Adjust</enum>
          </property>
          <property name="layoutMode">
           <enum>QListView::Batched</enum>
          </property>
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="gridSize">
           <size>
            <width>120</width>
            <height>100</height>
           </size>
          </property>
          <property name="viewMode">
           <enum>QListView::IconMode</enum>
          </property>
          <property name="uniformItemSizes">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="destImagePane">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget2">
        <layout class="QVBoxLayout" name="albumBlock">
         <item>
          <widget class="QLabel" name="albumLabel">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>12</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Album (drag pictures here)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="AlbumView" name="albumView">
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::DragDrop</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>160</width>
             <height>160</height>
            </size>
           </property>
           <property name="movement">
            <enum>QListView::Snap</enum>
           </property>
           <property name="resizeMode">
            <enum>QListView::Adjust</enum>
           </property>
           <property name="layoutMode">
            <enum>QListView::Batched</enum>
           </property>
           <property name="gridSize">
            <size>
             <width>180</width>
             <height>180</height>
            </size>
           </property>
           <property name="viewMode">
            <enum>QListView::IconMode</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget3">
        <layout class="QVBoxLayout" name="captionBlock" stretch="1,3,0">
         <item>
          <widget class="QLabel" name="originalPath">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>pathName</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="captionWidgets" stretch="1,10,1">
           <item>
            <widget class="QPushButton" name="back">
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="albumcreator_gui.qrc">
               <normaloff>:/:/images/1leftarrow.png</normaloff>:/:/images/1leftarrow.png</iconset>
             </property>
             <property name="shortcut">
              <string>Ctrl+Left</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="captionEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>48</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>48</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="forward">
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="icon">
              <iconset resource="albumcreator_gui.qrc">
               <normaloff>:/:/images/1rightarrow.png</normaloff>:/:/images/1rightarrow.png</iconset>
             </property>
             <property name="shortcut">
              <string>Ctrl+Space</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="cpationLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>12</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <underline>false</underline>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>Caption (press ctrl-SPC to move to next picture):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecently_edited">
     <property name="title">
      <string>Recently edited</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuRecently_edited"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCuttingTool"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuCreate">
    <property name="title">
     <string>Create</string>
    </property>
    <addaction name="actionCreate_Web_Slideshow"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuCreate"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCuttingTool"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="albumcreator_gui.qrc">
     <normaloff>:/:/images/filenew.png</normaloff>:/:/images/filenew.png</iconset>
   </property>
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="albumcreator_gui.qrc">
     <normaloff>:/:/images/fileopen.png</normaloff>:/:/images/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="albumcreator_gui.qrc">
     <normaloff>:/:/images/filesave.png</normaloff>:/:/images/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="albumcreator_gui.qrc">
     <normaloff>:/:/images/exit.png</normaloff>:/:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="albumcreator_gui.qrc">
     <normaloff>:/:/images/editundo.png</normaloff>:/:/images/editundo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="albumcreator_gui.qrc">
     <normaloff>:/:/images/editredo.png</normaloff>:/:/images/editredo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="albumcreator_gui.qrc">
     <normaloff>:/:/images/editcopy.png</normaloff>:/:/images/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCuttingTool">
   <property name="icon">
    <iconset resource="albumcreator_gui.qrc">
     <normaloff>:/:/images/editcut.png</normaloff>:/:/images/editcut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="albumcreator_gui.qrc">
     <normaloff>:/:/images/editpaste.png</normaloff>:/:/images/editpaste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About AlbumCreator...</string>
   </property>
  </action>
  <action name="actionCreate_Web_Slideshow">
   <property name="text">
    <string>Create Web Slideshow...</string>
   </property>
  </action>
  <action name="actionActionCuttingTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="albumcreator_gui.qrc">
     <normaloff>:/:/images/editcut.png</normaloff>:/:/images/editcut.png</iconset>
   </property>
   <property name="text">
    <string>actionCuttingTool</string>
   </property>
   <property name="toolTip">
    <string>Cut off margins or cut out a detail.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FileBrowserListView</class>
   <extends>QListView</extends>
   <header location="global">filebrowserlistview.h</header>
  </customwidget>
  <customwidget>
   <class>AlbumView</class>
   <extends>QListView</extends>
   <header location="global">albumview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="albumcreator_gui.qrc"/>
 </resources>
 <connections/>
</ui>
